setup.txt

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE tweets(tweet_id SERIAL PRIMARY KEY, user_name INTEGER, FOREIGN KEY(user_name) REFERENCES users(user_name), tweet VARCHAR(240), time TIMESTAMP);
```

Comment:
I tried following this, which looks very helpful to have, but I think it's in the wrong order because I get `ERROR: relation "users" does not exist` (relation means table)